pkg/config/dervied.go

```diff
@@ -34,6 +34,7 @@ func (idCfg *IdentityConfig) loadDerivedConfig() error {
 
 	// depends on the following:
 	// - derivedTargetDomainRoles()
+	// - derivedServiceCertConfig()
```

Comment:
Lets put this above the `derivedTargetDomainRoles()`

Reply:
fixed in: https://github.com/AthenZ/k8s-athenz-sia/pull/149/commits/41dd66096d1a4abd27b3d5bdb1fd1581f8d0007c